experiments/yarn/src/test/scala/io/gearpump/experiments/yarn/ContainerLaunchContextSpec.scala

```diff
@@ -29,10 +29,18 @@ class ContainerLaunchContextSpec extends FlatSpecLike with MockitoSugar {
   "A ContainerLaunchContext object" should "create new ContainerLaunchContext" in {
     val command = "foo"
     val hdfsRoot = "hdfs"
+    val mockVersion = "mockVersion"
     val yarnConfig = mock[YarnConfiguration]
     val appConfig = mock[AppConfig]
     when(appConfig.getEnv(Constants.HDFS_ROOT)).thenReturn(hdfsRoot)
+    when(appConfig.getEnv("version")).thenReturn(mockVersion)
+    when(yarnConfig.get("fs.defaultFS", "file:///")).thenReturn("file:///")
+    import sys.process._
+    s"mkdir $hdfsRoot".!
```

Comment:
You can use the `File` API and wrap it in a try-finally block for resource collection.